package.json

```diff
@@ -58,15 +59,9 @@
   "resolutions": {
     "**/tslib": "^2.3.1"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
   "lint-staged": {
     "*.{ts,tsx,js,jsx,json,md,mdx}": [
-      "prettier --write",
-      "git add"
+      "prettier --write"
```

Comment:
Not directly related to this change but we could expand `prettier` coverage by changing the lint-staged config a bit:

```json
"*": [
  "prettier --write --ignore-unknown"
]
```

`prettier` will format changes for any filetype it recognizes and ignores everything else.

Reply:
Can make a separate PR if maintainers would like to use this flag.

Reply:
cc. @Andarist